assets/js/templates/core_library_template.html

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+  <div class="row">
+    <h2 class="col-md-12">Library View - Core</h2>
+  </div>
+
+  <div class="row">
+    <div class="col-md-4">
+      <input type="text" placeholder="New Module" class="form-control" ng-model="name">
```

Comment:
I believe it's best practice to have the value in ng-model be an object that is intialized in the controller and you assign it a property here. ex. "ng-model=newModule.name" and in the controller "$scope.newModule = {}"